ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html

```diff
@@ -68,13 +68,13 @@ <h3>{{ _('Dataset Metadata') }}</h3>
             <tr>
                 <th scope="row" class="dataset-label">{{ field.label }}</th>
                 <td class="dataset-details">
-                    <span style="font-weight:bold">{{_('OthId Identifier')}}: </span><span>{{ alt_id.identifier }}</span><br/>
+                    <span style="font-weight:bold">{{_('Other Identifier')}}: </span><span>{{ alt_id.identifier }}</span><br/>
 
```

Comment:
these i18n changes were not requested... some label are now mismatching due to the fuzzy completion 
Opening #118